art/estimators/classification/keras.py

```diff
@@ -300,14 +300,18 @@ def _initialize_params(
         # Get the internal layer
         self._layer_names = self._get_layers()
 
-    def loss(self, x: np.ndarray, y: np.ndarray, **kwargs) -> np.ndarray:
+    def loss(self, x: np.ndarray, y: np.ndarray, reduction: str = "none", **kwargs) -> np.ndarray:
```

Comment:
Have you also considered using `reduction: Optional[str] = None` here?

Reply:
Yes, I considered it. The reason for `"none"` and the options for `reductions was that it is what PyTorch is using for its loss functions.

Reply:
That's a good argument 👍